bot/seasons/evergreen/bookmark.py

```diff
@@ -0,0 +1,53 @@
+import logging
+import random
+
+import discord
+from discord.ext import commands
+
+from bot.constants import Colours, ERROR_REPLIES
+
+log = logging.getLogger(__name__)
+
+
+class Bookmark(commands.Cog):
+    """A cog that creates personal bookmarks by relaying a message to the user's DMs."""
+
+    def __init__(self, bot: commands.Bot):
+        self.bot = bot
+
+    @commands.command(name="bookmark", aliases=("bm", "pin"))
+    async def bookmark(self, ctx: commands.Context, target_message: discord.Message, *,
+                       title: str = "Bookmark") -> None:
+        """Send you a link to the provided message in DM."""
+        log.info(f"{ctx.author} bookmarked {target_message.jump_url} with hints {title}.")
+
+        if not title:
+            title = "No hint provided."
+
+        embed = discord.Embed(
+            title=title,
+            colour=Colours.soft_green,
+            description=f"{target_message.content} \n\n[Give it a visit.]({target_message.jump_url})"
+        )
+        embed.set_author(name=target_message.author)
+        embed.set_author(name=target_message.author, icon_url=target_message.author.avatar_url)
+        embed.set_thumbnail(url="https://img.icons8.com/color/48/FF3333/"
+                            "bookmark-ribbon.png")
+        embed.set_footer(text=f"{ctx.author}")

```

Comment:
You're sending this bookmark to someone's DM; it is necessary to add their display name to the footer of the embed? My guess is that they'll probably remember who they are. That means that this doesn't add information to the embed and could be removed.